### ISSUE TITLE
Add character description page

### ISSUE THREAD

**Issue by Garados007** [Garados007/Werewolf#37]
Add a page that loads all character information with their description and present them in a simple and clear way.

- list all characters with their images, description and name among each other
- create "sub" pages for each game mode and themes
   - make for each sub page a new virtual url link that can be used to navigate directly to this section

**Comment by vangelif** [comment]
Can you by any chance give a step by step guide on setting-up and running both this repo & pronto? Thanks

**Comment by Garados007** [comment]
> Can you by any chance give a step by step guide on setting-up and running both this repo & pronto? Thanks

Yes, I had this in mind for a while but didn't do this, because I didn't expect someone using it. I thought, it should be enough in most cases to just use the existing infrastructure. I will relay further discussion for this topic into a new issue.
